src/Api/index.js

```diff
@@ -1,5 +1,6 @@
 import BASE_URL from './config';
 import axios from 'axios';
+import socketClient from 'socket.io-client';
```

Comment:
what is the purpose of this here

Reply:
I used it to connect to the socket server.

Reply:
sorry for  my incovenience